packages/core/admin/admin/src/components/GuidedTour/utils/setStateToLocaleStorage.js

```diff
@@ -0,0 +1,13 @@
+const setStateToLocaleStorage = currentStep => {
```

Comment:
Can you refer to my comment above and merge this into the object that will manage the localStorage?